test_tracing_with_deployment.py

```diff
@@ -0,0 +1,33 @@
+from starlette.requests import Request
```

Comment:
Added this example to show that on all the serve deployments we have to do tracing injection and see the comment below.